docs/blog/posts/introducing-structured-outputs.md

```diff
@@ -0,0 +1,468 @@
+---
+draft: False
+date: 2024-08-20
+slug: is-instructor-dead
+tags:
+  - OpenAI
+authors:
+  - ivanleomk
+---
+
+# Is Instructor Dead?
+
+## What's Open AI's Structured Output mode all about?
+
+OpenAI's new Structured Output mode is a huge step change for developers building complex workflows. Given an arbitrary JSON Schema, Structured Output ensures that the response matches the schema exactly.
```

Comment:
1. mention that we were referenced to the release article
2. link to the release article